packages/app/src/contexts/Web3Context/Web3Context.tsx

```diff
@@ -40,6 +41,7 @@ export const Web3ContextProvider: React.FC = ({ children }) => {
         } else {
             await connector.killSession();
             await connectDID(connector, wallet);
+            await useBurner();
```

Comment:
This is not giving us back the burner wallet. We need to return the burner wallet from the context